scripts/entrypoint.sh

```diff
@@ -36,9 +36,13 @@ if [ ! -f grpc-health-probe ]; then
     exit 1
 fi
 
-AGENT_LOG_PATH=${AGENT_LOG_PATH:-aws-k8s-agent.log}
-HOST_CNI_BIN_PATH=${HOST_CNI_BIN_PATH:-/host/opt/cni/bin}
-HOST_CNI_CONFDIR_PATH=${HOST_CNI_CONFDIR_PATH:-/host/etc/cni/net.d}
+AGENT_LOG_PATH=${AGENT_LOG_PATH:-"aws-k8s-agent.log"}
+HOST_CNI_BIN_PATH=${HOST_CNI_BIN_PATH:-"/host/opt/cni/bin"}
+HOST_CNI_CONFDIR_PATH=${HOST_CNI_CONFDIR_PATH:-"/host/etc/cni/net.d"}
+AWS_VPC_K8S_CNI_VETHPREFIX=${AWS_VPC_K8S_CNI_VETHPREFIX:-"eni"}
+AWS_VPC_ENI_MTU=${AWS_VPC_ENI_MTU:-"9001"}
+AWS_VPC_K8S_PLUGIN_LOG_FILE=${AWS_VPC_K8S_PLUGIN_LOG_FILE:-"/var/log/aws-routed-eni/plugin.log"}
+AWS_VPC_K8s_PLUGIN_LOG_LEVEL=${AWS_VPC_K8S_PLUGIN_LOG_LEVEL:-"Debug"}
```

Comment:
Nit: `AWS_VPC_K8s_PLUGIN_LOG_LEVEL` here and on line 85 still has the lower case `s`.